examples/ui-Text.ts

```
@@ -0,0 +1,88 @@
+    // Add text renderer for text entity
+    const text = entity.addComponent(Text);
+    // Set text color
+    text.color = color;
+    // Set text to render
+    text.text = "The quick brown fox jumps over the lazy dog";
+    // Set font with font family
+    text.font = Font.createFromOS(entity.engine, fontFamily);
+    // Set font size
+    text.fontSize = fontSize;
+    // Set font whether bold
+    bold && (text.fontStyle |= FontStyle.Bold);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)